src/amo/pages/Feedback/index.js

```diff
@@ -0,0 +1,478 @@
+                        'For allegations of trademark or copyright infringement, I further ' +
+                        'certify that I am either the affected rightsholder, or have been ' +
+                        "authorized to act on that rightsholder's behalf.",
+                    )}
+                  </label>
+                </Card>
+
+                <Card
+                  className="Feedback--Card"
+                  header={i18n.gettext('Your details')}
+                >
+                  <p className="Feedback-details-aside">
+                    {currentUser
+                      ? i18n.gettext(
+                          'We will notify you by email with any updates about your report.',
+                        )
+                      : i18n.gettext(
+                          'To receive updates about your report enter your name and email address.',
+                        )}
+                  </p>
+                  <label className="Feedback--label" htmlFor="feedbackName">
+                    {i18n.gettext('Your Full Name')}
+                  </label>
+                  <input
+                    className="Feedback-name"
+                    id="feedbackName"
+                    name="name"
+                    disabled={currentUser}
+                    onChange={this.onFieldChange}
+                    value={this.state.name || (currentUser && currentUser.name)}
+                  />
+
+                  <label className="Feedback--label" htmlFor="feedbackEmail">
+                    {i18n.gettext('Your Email Address')}
+                  </label>
+                  <input
+                    className="Feedback-email"
+                    id="feedbackEmail"
+                    name="email"
+                    disabled={currentUser}
+                    onChange={this.onFieldChange}
+                    value={
+                      this.state.email || (currentUser && currentUser.email)
+                    }
+                  />
+                </Card>
+
+                <div className="Feedback-buttons-wrapper">
+                  <Button
+                    buttonType="action"
+                    className="Feedback-submit-button Feedback-button"
+                    disabled={this.preventSubmit()}
+                    puffy
+                    type="submit"
+                  >
+                    {submitButtonText}
+                  </Button>
+                </div>
+              </div>
+            </form>
+          )}
+        </div>
+      </Page>
+    );
+  }
+}
+
+function mapStateToProps(
+  state: AppState,
+  ownProps: InternalProps,
+): PropsFromState {
+  const { params } = ownProps.match;
+  const { addonIdentifier } = params;
+
+  const currentUser = getCurrentUser(state.users);
+
+  return {
+    currentUser,
+    loading: state.abuse.loading,
+    addonIdentifier,
+    abuseReport: state.abuse.byGUID[addonIdentifier]
+      ? state.abuse.byGUID[addonIdentifier]
```

Comment:
What if we pass something else than the `guid` in `addonIdentifier`? If this page only accepts the add-on GUID, then it's URL parameter should be named `:guid`.

Reply:
The underlying api would accept either a numeric add-on id, a slug, or a guid, but the state in frontend differentiates these when storing data so I guess we need it to be a guid because guid is the lowest common denominator

Reply:
The abuse report API will accept add-on identifier used elsewhere in the api, so numeric add-on id, slug, or guid.  We'll load the add-on in https://github.com/mozilla/addons-frontend/issues/12478 but at this point any extra loading seems unnecessary.